src/client/Microsoft.Identity.Client/Internal/Requests/PublicAuthCodeRequest.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client.ApiConfig.Parameters;
+using Microsoft.Identity.Client.Http;
+using Microsoft.Identity.Client.TelemetryCore.Internal.Events;
+using Microsoft.Identity.Client.OAuth2;
+using Microsoft.Identity.Client.UI;
+
+namespace Microsoft.Identity.Client.Internal.Requests
+{
+    internal class PublicAuthCodeRequest : RequestBase
```

Comment:
Second part of the InteractiveRequest class - again, 1 ctor, 1 public method, everything readonly.